docs/Validation-and-Serialization.md

```diff
@@ -109,16 +109,28 @@ The function `getSchemas` returns all shared schemas that were added by `addSche
 
 The `schemaCompiler` is a function that returns a function that validates the body, url parameters, headers, and query string. The default `schemaCompiler` returns a function that implements the `ajv` validation interface. Fastify uses it internally to speed the validation up.
 
-While you cannot change the configuration options of the default `ajv` instance, you can create your own:
+The configuration options of the default fastify `ajv` instance are set as
```

Comment:
"fastify" => "Fastify"

Reply:
In fact, just replace the sentence with:

"Fastify's baseline `ajv` configuration is:"